### PR TITLE
BZ#2075083 add appendix to SHE Install for allowed CPU types

### PR DESCRIPTION
**Pull request by emarcusRH**
Signed-off-by: emarcusRH <emarcus@redhat.com>

**Downstream only - applies to RHV 4.4 SP1** 
preview: 
http://file.tlv.redhat.com/emarcus/2075083/#proc-proc-Defining_allowed_CPU_migrate_DWH_DB

Fixes issue # | [BZ#2075083](https://bugzilla.redhat.com/show_bug.cgi?id=2075083)  
Changes proposed in this pull request:

- add appendix to SHE Install for allowed CPU types

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/main/CONTRIBUTING.md): (please @emarcusRH )

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

File: source/documentation/common/install/proc-Defining_allowed_CPU.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+[id="proc-proc-Defining_allowed_CPU_{context}"]
+= Defining allowed CPU types in Self-hosted Engine deployment
+
+[role="_abstract"]
+
+
+
+.Procedure
+
+. Create a file named deploy.json, and provide a CPU type name for the value `cpu_type_name` from the table shown below. For example, if the CPU type you want is *Secure Intel Nehalem Family*, then the `deploy.json` should look like the following:
```
Comment (RichardHoch) [inline source/documentation/common/install/proc-Defining_allowed_CPU.adoc:11]: There should be backticks around deploy.json, no?

File: source/documentation/common/install/proc-Defining_allowed_CPU.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
++
+----
+[root@host ~]# hosted-engine --deploy --ansible-extra-vars=@/root/deploy.json
+----
+
+[id="Allowed_CPU_types"]
+.Allowed CPU Types
+[options="header"]
+|===
+|CPU type name |CPU properties
```
Comment (RichardHoch) [inline source/documentation/common/install/proc-Defining_allowed_CPU.adoc:30]: Same with the CPU properties.

Comment (emarcusRH) [reply source/documentation/common/install/proc-Defining_allowed_CPU.adoc:30]: in this case, it's just a generic column title, not the name of a variable.

**Review by RichardHoch — APPROVED** [review]

**Comment by RichardHoch** [comment]
lgtm

**Pull request merged**